build.gradle

```diff
@@ -128,6 +128,7 @@ subprojects { subproject ->
 		mysqlVersion = '6.0.6'
 		pahoMqttClientVersion = '1.2.0'
 		postgresVersion = '42.0.0'
+		powermockVersion = '1.7.1'
```

Comment:
Redundant variable.

Anyway keep in mind that we are not going to use `Power Mock` at all.